src/swbf_fnv_hashes.cpp

```diff
@@ -1,2834 +1,3935 @@
 
 #include "swbf_fnv_hashes.hpp"
 #include "synced_cout.hpp"
-
 #include <cstdint>
+#include <fstream>
+#include <sstream>
+#include <iostream>
+#include <mutex>
+#include <regex>
 #include <string>
```

Comment:
Also include shared_mutex so we can use it instead of `std::mutex` and avoid blocking when multiple threads are just reading from `swbf_hashes`.
```suggestion
#include <shared_mutex>
#include <string>
```